baselines/fedmeta/fedmeta/conf/data/shakespeare.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# this is the config that will be loaded as default by main.py
+# Please follow the provided structure (this will ensuring all baseline follow
+# a similar configuration structure and hence be easy to customise)
+
+model:
+  _target_: models.StackedLSTM
+
+client_resources:
+  num_cpus: 4
+  num_gpus: 1.0
```

Comment:
```suggestion
  num_cpus: 4
  num_gpus: 0.2
```

Reply:
I think these clients do not need an entire GPU (VRAM usage < 1GB). Or did you experience some problems using a `num_gpus` lower than 1?

Reply:
Hi, @jafermarq. For both the Femnist and Shakespeare datasets, basic federated learning (FedAvg and FedAvg_Meta) can be trained using less than one GPU. However, during my experiment, when meta-learning was incorporated, many clients experienced out-of-memory errors during the second derivative computation of the LSTM model.